### DISCUSSION TITLE
How to show commands or menu in telegram interface?

### DISCUSSION THREAD

**Discussion by maksakoviliya** [telegram-bot-sdk/telegram-bot-sdk#101] [Q&A]
How to show something like this in native telegram interface?
<img width="430" alt="image" src="https://github.com/telegram-bot-sdk/telegram-bot-sdk/assets/16371638/2e736ad2-ab53-4018-a106-63f9cd455d6e">

**Comment by swim89** [comment]
You have to register command with BotFather bot with command `/setcommands`
